### ISSUE TITLE
Train model on small dataset

### ISSUE THREAD

**Issue by ADAMS12345678** [autonomousvision/stylegan_xl#81]
Hello author!
I train an unconditional class image generation model on a small dataset (under 100 images), however the model does not generate multiple types of images, no matter how much the seed is set, it is only a small variation of a single type of image. May I ask what is the reason for this? I would be very grateful for your answer!

**Comment by nom57** [comment]
check your stats.jsonl

either diffaug doesnt work or you have a "NaN" value for your scores/fake , or you could have misconfigured batch size and learning rate.
